app/models/user.rb

```diff
@@ -1,18 +1,37 @@
 # This will handle user info via the devise gem
 class User < ActiveRecord::Base
   has_many :wikis, dependent: :destroy
-  after_initialize :set_user_default, if: :new_record?
+  after_initialize :set_default_role, if: :new_record?
 
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable, :confirmable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  enum role: [:admin, :standard, :premium]
+  enum role: [:standard, :premium, :admin]
+
+  # Method to handle upgrading user
+  def upgrade_user
+    set_role_to_premium
+  end
+
+  # Method to handle downgrading user
+  def downgrade_user
+    set_role_to_standard
+  end
 
   private
 
-  def set_user_default
-    self.role = 'standard'
+  def set_default_role
```

Comment:
Hmm, I thought I took that out... its a fragment because I decided to get rid of this and call a different method to set the default.